packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -92,23 +92,17 @@ const injectRscPayload = (
 ) => {
   const chunks: Uint8Array[] = [];
   let closed = false;
-  let notify: (() => void) | undefined;
   const copied = readable.pipeThrough(
     new TransformStream({
       transform(chunk, controller) {
         if (!(chunk instanceof Uint8Array)) {
           throw new Error('Unknown chunk type');
         }
-        controller.enqueue(chunk);
         chunks.push(chunk);
-        // HACK delay a little bit to avoid hydration mismatch
-        setTimeout(() => {
-          notify?.();
```

Comment:
I thought it's better to push as quickly as possible, but hydration process seems sensitive with the order, so it's safer to always send it after html. Not sure if it causes undesirable behavior in edge cases.